multinet/api/views/workspace.py

```diff
@@ -49,6 +49,21 @@ def create(self, request):
         assign_perm('owner', request.user, workspace)
         return Response(WorkspaceSerializer(workspace).data, status=status.HTTP_200_OK)
 
+    @swagger_auto_schema(
+        request_body=WorkspaceRenameSerializer(),
+        responses={200: WorkspaceSerializer()},
+    )
+    @method_decorator(permission_required_or_403('owner', (Workspace, 'name', 'name')))
```

Comment:
A general question for @AlmightyYakob and @naglepuff--I'm surprised to see stringly-typed role names here, rather than symbols defined in an enum. Did I miss the boat on a newer way of doing things?

Reply:
Not sure when this branch was created but it is behind master. I don't think it has any of the new permissions stuff in it. I'm not a git expert but I think a rebase would be good here.

Reply:
A merge in this case would be better than a rebase (since reviewing has already begun).

@eagw, when you do merge master into your branch, you may run into merge conflicts. If you do, feel free to ask me or Jake for help in resolving them.

Reply:
It's now merged and the new permissions model has been used.

See 78d4b3f and 3fbfbc3